cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
@jdemon519 so you need to look at what images are being returned by both wrapanapi and CFME. Obviously this didn't run on PRT so I can't debug, but the cfme.log should also show you the mismatch, ie.....```3 != 4``` or something like that. Then you need to find out if we need to modify wrapanapi to produce the right numbering/listing for CFME.